src/Server/RestRouterHandlers/RestRouterHandler.h

```diff
@@ -119,8 +119,8 @@ class RestRouterHandler : private boost::noncopyable
 
     String processQuery(
         const String & query,
-        const std::function<void(Block &&)> & callback = [](Block &&) {},
-        const Poco::JSON::Object & resp = Poco::JSON::Object()) const;
+        const Poco::JSON::Object & resp = Poco::JSON::Object(),
```

Comment:
Let's accept `Poco::JSON::Object & resp`.  We can have overloaded `processQuery` functions if necessary. For example, if sometimes we don't care `resp`, we can have `processQuery(const String & query, std::function<void(Block &&)> callback)`; if user don't care `callback`, then we can have a different one ...

Reply:
fixed